Update/gitlab integration page with a troubleshoot section (#470)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

To connect your personal GitLab account, navigate to your [User Profile](https://cloud.getdbt.com/#/profile/) select the [Integrations section](https://cloud.getdbt.com/#/profile/integrations/) from the left sidebar. Under "GitLab", if your account is already connected, you’ll see "Your user account is linked to a GitLab account with username &lt;your-gitlab-username&gt;."

If your account is not connected, you’ll see "This account is not linked to a GitLab account.” Click the button to begin the setup process. You’ll be redirected to GitLab, and then back into dbt Cloud. When you are redirected to dbt Cloud, you should now see your connected account. 

## Troubleshooting

> ### Respositories Errors

Reviewer: ```suggestion
### Errors importing a repository
```

If you do not see your repository listed, double-check that:

> - You are a maintainer on that repository (only users with a maintainer or owner permissions can set up repository connections

Reviewer: I love that you added this point, but I think the failure mode is actually different. You will see the repo, but you should get an error when dbt Cloud attempts to create a Deploy Token in the repo. The repo should show up though

Author: ah okay, updating it now.

To connect your personal GitLab account, navigate to your [User Profile](https://cloud.getdbt.com/#/profile/) select the [Integrations section](https://cloud.getdbt.com/#/profile/integrations/) from the left sidebar. Under "GitLab", if your account is already connected, you’ll see "Your user account is linked to a GitLab account with username &lt;your-gitlab-username&gt;."

If your account is not connected, you’ll see "This account is not linked to a GitLab account.” Click the button to begin the setup process. You’ll be redirected to GitLab, and then back into dbt Cloud. When you are redirected to dbt Cloud, you should now see your connected account. 

## Troubleshooting

### Errors importing a repository
If you do not see your repository listed, double-check that:
- Your repository is in a Gitlab group you have access to

> If you do see your repository listed but after selecting it, it's stuck loading/errors, double-check that:

Reviewer: ```suggestion
If you do see your repository listed but are unable to import the repository successfully, double-check that:
```

### Errors importing a repository
If you do not see your repository listed, double-check that:
- Your repository is in a Gitlab group you have access to

If you do see your repository listed but after selecting it, it's stuck loading/errors, double-check that:

> - You are a maintainer on that repository (only users with a maintainer or owner permissions can set up repository connections

Reviewer 2: missing a ")" at the end of this line. 

Other than that, looks great to me!

Author: That's so weird...I see it?

Author: I just fixed it so it's possible you were on an older page?
